src/site/apt/user_query_hdfs.apt

```diff
@@ -0,0 +1,131 @@
+~~ contributor license agreements.  See the NOTICE file distributed with
+~~ this work for additional information regarding copyright ownership.
+~~ The ASF licenses this file to You under the Apache License, Version 2.0
+~~ (the "License"); you may not use this file except in compliance with
+~~ the License.  You may obtain a copy of the License at
+~~
+~~     http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing, software
+~~ distributed under the License is distributed on an "AS IS" BASIS,
+~~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~~ See the License for the specific language governing permissions and
+~~ limitations under the License.
+
+Executing a Query in HDFS
+
+
+* 1. Connecting VXQuery with HDFS
+
+    The only variable VXQuery needs in order to access HDFS is the path to the HDFS configuration directory.
+
+    We define that path in the <<cluster.properties>> file that is located in <<vxquery-root/vxquery-server/src/resources/conf/>> directory.
+
+    Inside that file we need to change the value of the variable <<HDFS_CONF>> to the full system path of the HDFS configuration.
+
+    For example, <HDFS_CONF=/usr/local/hadoop/etc/hadoop> defines that the configuration files for our HDFS installation are located in </usr/local/hadoop/etc/hadoop> path.
+
+
+* 2. Running the Query
+
+  For files stored in HDFS there are 2 ways to access them from VXQuery.
+
+  [[a]] Reading them as whole files.
+
+  [[b]] Reading them block by block.
+
+** a. Reading them as whole files.
+
+  For this option you only need to change the path to files.To define that your file(s) exist and should be read from HDFS you must add <"hdfs:/"> in front of the path.VXQuery will read the path of the files you request in your query and try to locate them.
+
+  So in order to run a query that will read the input files from HDFS you need to make sure that
+
+  a) The configuration path is correctly set to the one of your HDFS system.
+
+  b) The path defined in your query begins with <hdfs:/> and the full path to the file(s).
+
+  c) The path exists on HDFS and the user that runs the query has read permission to these files.
+
+*** Example
+
+  I want to find all the <books> that are published after 2004.
+
+  The file is located in HDFS in this path </user/hduser/store/books.xml>
+
+  My query will look like this:
+
+----------
+for $x in collection("hdfs://user/hduser/store")
+where $x/year>2004
+return $x/title
+----------
+
+  If I want only one file, the <<books.xml>> to be parsed from HDFS, my query will look like this:
+
+----------
+for $x in doc("hdfs://user/hduser/store/books.xml")
+where $x/year>2004
+return $x/title
+----------
+
+** b. Reading them block by block
+
+  In order to use that option you need to modify your query.Instead of using the <collection> or <doc> function to define your input file(s) you need to use <collection-with-tag>.
```

Comment:
Make sure you have spaces between your sentences. Also instead of writing long lines, put returns so they do not wrap (80 or 100 characters per line). I usually leave a word space gap at the end just in case later we edit the line it does not need reformatting.